etcdserver/etcdserverpb/rpc.proto

```diff
@@ -510,6 +510,9 @@ message Compare {
     // value is the value of the given key, in bytes.
     bytes value = 7;
   }
+  // range_end compares over the range [key, range_end). See RangeRequest.
+  bytes range_end = 8;
```

Comment:
Yes, it's the same target; will clarify comment

An example use case from the leasing kv: delete range. To safely delete a range of keys the leasing kv must ensure no other client holds a key lease (represented as a "leasing key" controlled by the leasing protocol) on any key in the range. The leasing kv can acquire/revoke key leases such that so long as there are no new key leases acquired on a range, that range is safe to delete. This check can be done with a ranged comparison over the corresponding leasing keys and testing `CreateRev < safeToDeleteRev`.